lib/ui/screens/UserInfoCollectorScreen.dart

```diff
@@ -78,155 +80,40 @@ class _UserInfoCollectorScreenState extends BaseState<UserInfoCollectorScreen> {
-                    text: ' Privacy Policy.',
-                    recognizer: new TapGestureRecognizer()
-                      ..onTap = () {
-                        launchWebUrl(PRIVACY_POLICY);
-                      },
-                    style: new TextStyle(fontWeight: FontWeight.bold)),
-              ],
-            ),
-          ),
-          alignment: Alignment.center,
-        ),
-        SizedBox(
-          height: 20,
-        ),
-      ],
-    );
+  Widget firstCardContent() {
+    Widget questionPair = CTQuestionPair(
+        positiveQuestionText: "I am not infected with Coronavirus",
+        negativeQuestionText: "I am infected with Coronavirus",
+        onNegativeQuestionClick: () async {
+          dialogOnResponse(2);
+          FirestoreRepository.setUserSeverity(1);
+        },
+        onPositiveQuestionClick: () async {
+          dialogOnResponse(2);
+          FirestoreRepository.setUserSeverity(0);
+        });
+
+    return CTBottomSheetWidget(
+        mainQuestionText: "Q. How are you feeling today?",
+        subSectionDescription:
+            "Don't worry. No personal information will be stored. Your location will be tracked, but all data will be anonymous.",
+        questionPairWidget: questionPair,
+        onTermsConditionsClick: (String key) async {
+          showLoadingDialog(tapDismiss: false);
+          String url = await FirestoreRepository.getRemoteConfigValue(key);
+          print(url);
+          hideLoadingDialog();
+          if (await canLaunch(url)) {
+            await launch(url);
+          }
+        });
   }
 
-  Column secondCardContent() {
-    return Column(
-      crossAxisAlignment: CrossAxisAlignment.start,
-      children: <Widget>[
-        Text(
-          "Thank you for your response!",
-          style: TextStyle(fontSize: 26, fontWeight: FontWeight.bold),
-        ),
-        SizedBox(
-          height: 10,
-        ),
-        Text(
-          "Thank you for joining hands and helping humanity. Remember you can always comeback and submit an updated status.",
-          style: TextStyle(fontSize: 16),
-        ),
-        SizedBox(
-          height: 160,
-        ),
-        Align(
-          child: SizedBox(
-            width: double.infinity,
-            height: 50,
-            child: MaterialButton(
-              color: Colors.white,
-              shape: RoundedRectangleBorder(
-                borderRadius: new BorderRadius.circular(8.0),
-                side: BorderSide(color: Colors.black),
-              ),
-              onPressed: () {
-                dialogOnResponse(1);
-              },
-              child: Text(
-                "Got It",
-                style: TextStyle(color: Colors.black, fontSize: 18),
-              ),
-            ),
-          ),
-          alignment: Alignment.center,
-        )
-      ],
+  Widget secondCardContent() {
+    return CTThankYouDialog(
+      onButtonClick: () {
+        dialogOnResponse(1);
```

Comment:
@SamIAmHarris can we rename the function name to switchScreen, I know dialogOnResponse doesn't sound right.